scvi/utils/_decorators.py

```diff
@@ -0,0 +1,26 @@
+from functools import wraps
+from typing import Callable
+
+from anndata import AnnData
+from mudata import MuData
+
+from scvi.data._utils import _get_latent_adata_type
+
+
+def unsupported_in_latent_mode(fn: Callable) -> Callable:
```

Comment:
if we create an extended version of `BaseModelClass` with some latent mode functionality we can add a property to the model that this decorator can then use to check if it's supported